### PR TITLE
Allow newlines after identifier

### PR DESCRIPTION
**Pull request by fsmiamoto**
Related to #189

Hello there! Thanks for the awesome tool.

Saw that there was an issue about newlines after the `const` identifier and decided to give it a try to contribute to
the project.

I'm not really familiar with the internals so let me know if the approach I took is the right one.

There's also a unit test failing and any advice on how to approach fixing it would be great.

### CODE REVIEW

**Comment by jaqx0r** [comment]
Thanks for your help!  I'll let the workflows run then see what the failing test is.

**Comment by jaqx0r** [comment]
The unit test failure is parser_test.go:615; it's a table driven test that is checking what the syntax tree thinks the position is when it parses a const pattern definition, and the result has changed because of your edits.  It looks like you need to update the test data to change the column number.  I need to think if that's expected or not, though...

**Comment by jaqx0r** [comment]
Yeah I think the new value is correct, it's suppose to show you the positino of the literal `/.../` so starting at column 6 was incorrect.  Adding the `opt_nl` has reset the position marker, fixing the bug.

What I'd also like to see is a copy of your new multiline test in that position test to see if the position is correctly computed for the multiline literals.

**Comment by fsmiamoto** [comment]
Awesome! I've added an additional test case for the multiline regex position parsing and the results were not quite what I expected:
```
--- FAIL: TestParsePositionTests (0.00s)
    --- FAIL: TestParsePositionTests/multiline_regex (0.00s)
        parser_test.go:635: Unexpected diff, -want +got:
              []*position.Position{
                &{
                        Filename: "multiline regex",
                        Line:     1,
            -           Startcol: 0,
            +           Startcol: 4,
                        Endcol:   4,
                },
                &{Filename: "multiline regex", Line: 2, Endcol: 4},
              }
        parser_test.go:635: expected:
            [%!V(*position.Position=&{multiline regex 1 0 4}) %!V(*position.Position=&{multiline regex 2 0 4})]
        parser_test.go:635: received:
            [%!V(*position.Position=&{multiline regex 1 4 4}) %!V(*position.Position=&{multiline regex 2 0 4})]
```
I expected the position of the first node to be at `1 0 4` but it is currently resulting in `1 4 4` so my current guess is that we have a problem here.
If it makes sense, I'll try to dive deeper to find out why we're not getting the expected result.

File: internal/runtime/compiler/parser/parser_test.go [hunk 1]
```diff
@@ -605,7 +612,14 @@ var parsePositionTests = []struct {
...
 		`const ID /foo/`,
-		[]*position.Position{{"pattern", 0, 6, 13}},
+		[]*position.Position{{"pattern", 0, 9, 13}},
+	},
+	{
+		"multiline regex",
+		"const ID\n" +
+			"/foo/ +\n" +
+			"/bar/",
+		[]*position.Position{{"multiline regex", 1, 0, 4}, {"multiline regex", 2, 0, 4}},
```
[the remark this answers is not in the document]
Comment (fsmiamoto) [reply internal/runtime/compiler/parser/parser_test.go]: Yup, done!

**Review by jaqx0r — APPROVED** [review]

**Comment by jaqx0r** [comment]
Thanks!

**Pull request merged**